Purchases/RCBackend.m

```diff
@@ -214,6 +221,19 @@ - (void)postReceiptData:(NSData *)data
         body[@"subscription_group_id"] = subscriptionGroup;
     }
 
+    if (@available(iOS 12.2, macOS 10.14.4, *)) {
+        if (discounts) {
+            body[@"offers"] = [NSMutableArray array];
+            for (RCPromotionalOffer * discount in discounts) {
+                [body[@"offers"] addObject:@{
```

Comment:
Very minor legibility thing, but, I think creating an array in this block then setting it on the body after the loops is clearer than referencing a mutable in an `NSDictionary`.